Purchases/Error Handling/DescribableError.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  DescribableError.swift
+//
+//  Created by Joshua Liebowitz on 10/28/21.
+
+import Foundation
+
+protocol DescribableError: Error {
```

Comment:
Isn't this the same as `CustomStringConvertible`?

Reply:
It is! Updating 😄